Client/src/index.css

```diff
@@ -11,6 +11,27 @@
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+  --size-40: 40px;
+  --size-38: 38px;
+  --size-35: 35px;
+  --size-28: 28px;
+  --size-24: 24px;
+  --size-20: 20px;
+  --size-16: 16px;
+  --size-13: 13px;
+
+  --font-size-1: 13px;
+  --font-size-2: 1rem;
```

Comment:
Not a fan of mixed units, this is on us though, not on your refactoring :joy:  We should fix this at some point

Reply:
Same here lol, I prefer rem/em values but I agree, I feel this will be updated shortly. I'll await further review